spec/data-model/README.md

```diff
@@ -22,39 +22,43 @@ Two equivalent definitions of the data model are also provided:
 
 ## Messages
 
-A `SelectMessage` corresponds to a syntax message that includes _selectors_.
-A message without _selectors_ and with a single _pattern_ is represented by a `PatternMessage`.
+A `SimpleMessage` corresponds to a syntax message that includes _selectors_.
+A message without _selectors_ and with a single _pattern_ is represented by a `ComplexMessage`.
 
 In the syntax,
-a `PatternMessage` may be represented either as a _simple message_ or as a _complex message_,
+a `ComplexMessage` may be represented either as a _simple message_ or as a _complex message_,
```

Comment:
I think it's confusing to use "complex" to mean two different things (in the data model, having only a single pattern, regardless of declarations; in the concrete syntax, having no declarations and no selectors.)

I don't love the terms `PatternMessage` and `SelectMessage`, but I agree with Eemeli that it's probably best to keep it that way.